packages/ui/src/components/style/mixins/table.less

```diff
@@ -6,6 +6,7 @@
   text-align: left;
   border: @condo-global-border-width-default solid @condo-global-color-gray-3;
   border-radius: @condo-global-border-radius-medium;
+  border-collapse: separate !important;
```

Comment:
using !important is antypattern. instead you can increase specificity of the selector